### PR TITLE
Rebrand Griffon to Assurance

### PR DESCRIPTION
**Pull request by PravinPK**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/AEPAssuranceTestApp.Android/mono_crash.11a6748113.0.json [hunk 1]
```diff
@@ -0,0 +1,725 @@
+{
```
Comment (rymorale) [inline src/AEPAssuranceTestApp.Android/mono_crash.11a6748113.0.json:1]: this file should be removed / git ignored.

File: src/AEPAssuranceTestApp.iOS/mono_crash.11a6748113.0.json [hunk 2]
```diff
@@ -0,0 +1,731 @@
+{
```
Comment (rymorale) [inline src/AEPAssuranceTestApp.iOS/mono_crash.11a6748113.0.json:1]: another crash log to remove and gitignore

**Review by rymorale — APPROVED** [review]
looks good overall, just have a couple crash logs to cleanup and gitignore so they don't get added in the future.

**Comment by IGx89** [comment]
Was this completed? I'm seeing https://www.nuget.org/packages/Adobe.AEPAssurance.Android/ exists, uploaded around the same time as this PR.

**Pull request merged**